resources/schemas/dbscripts/postgresql/cds-23.000-23.001.sql

```diff
@@ -0,0 +1,95 @@
+CREATE TABLE cds.import_assay_combined_antigen_metadata
+(
+    antigen_cds_id VARCHAR(250),
```

Comment:
Make `antigen_cds_id` non-nullable since it's part of the compound PK.

Reply:
done